src/js/components/tour/plan.tsx

```diff
@@ -23,32 +28,48 @@ const steps: Step[] = [
   },
 ];
 
-const PlanTour = ({ run, handleCallback }: Props) => (
-  <Joyride
-    steps={steps}
-    run={run}
-    locale={{
-      back: i18n.t('Back'),
-      close: i18n.t('Close'),
-      last: i18n.t('Close'),
-      next: i18n.t('Next'),
-    }}
-    continuous
-    disableScrolling
-    callback={handleCallback}
-    styles={{
-      options: {
-        textColor: '#080707',
-        primaryColor: '#0070d2',
-      },
-      tooltipContainer: {
-        textAlign: 'left',
-      },
-    }}
-    floaterProps={{
-      disableAnimation: true,
-    }}
-  />
-);
+const PlanTour = ({ run, handleCallback }: Props) => {
+  const isMounted = useIsMounted();
+  const dispatch = useDispatch<ThunkDispatch>();
+
+  const doOnboard = useCallback(
+    (data: CallBackProps) => {
+      dispatch(onboard()).finally(() => {
+        /* istanbul ignore else */
+        if (isMounted.current) {
+          handleCallback(data);
+        }
+      });
+    },
+    [dispatch, isMounted, handleCallback],
+  );
```

Comment:
Like I said in my review comments yesterday, I think we want this "onboard" action to run immediately when the landing modal initially opens, not after the onboarding is done.